lib/util/ajv.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @fileoverview The instance of Ajv validator.
+ * @author Evgeny Poberezkin
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const Ajv = require("ajv"),
+    metaSchema = require("ajv/lib/refs/json-schema-draft-04.json");
```

Comment:
Ajv supports draft-06 by default - this code is needed as the schemas for all rules would be draft-04 and there are changes in draft-06 that would break them. So I thought that migrating to draft-06 would require a major version change as some third party rules will need to change their schemas.